### PR TITLE
kill COMMCAREHQ global

### PR DESCRIPTION
**Pull request by orangejenny**
@millerdev

### CODE REVIEW

File: corehq/apps/style/static/style/js/main.js
Comment (millerdev) [inline corehq/apps/style/static/style/js/main.js]: Would it be easy to turn these into classes with nice names like `hq-grip`? It's hard to remember all those classes, and the symbolic names were nice.

**Review by millerdev — APPROVED** [review]
Wow, excellent!

File: corehq/apps/style/static/style/js/main.js
Comment (orangejenny) [inline corehq/apps/style/static/style/js/main.js]: There isn't a good way to do it in LESS, because we run LESS only on HQ's styles, so we can't access the fontawesome classes like `fa`. But I'm fine with putting the constants back: https://github.com/dimagi/commcare-hq/pull/17400/commits/c6c545eff52a6a7d8e38375e2f7227bcefb8c39c

**Review by millerdev — APPROVED** [review]

**Comment by orangejenny** [comment]
test failure is just v2 diffs, I'lll run `build_app_manager_v2_diffs` now

**Comment by orangejenny** [comment]
@NoahCarnahan can you sanity check the last commit (https://github.com/dimagi/commcare-hq/pull/17400/commits/a054632fd55954785b4b264ea5a853d84e65f25a)? After that, this should be good to merge.

**Comment by NoahCarnahan** [comment]
a054632 looks good!

**Comment by orangejenny** [comment]
fixed a bunch of merge conflicts and sanity checked; will merge once tests pass

**Comment by millerdev** [comment]
```
FAIL: corehq.apps.userreports.tests.test_report_builder_v2_diffs:TestReportBuilderV2Diffs.test_diffs
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_report_builder_v2_diffs.py", line 102, in test_diffs
    f, filename_v1, filename_v2
AssertionError: 
*************************************
REPORT BUILDER V2 Diff Failure
An edit made to a V1 Report Builder file "forms.py" does not match the stored diff
of its V2 counterpart.
**Please make the edits to the V2 file so that it gets the changes from V1.**
Once you have done this, run ./manage.py build_report_builder_v2_diffs
to rebuild the broken diffs.
These files are located in
/mnt/commcare-hq-ro/corehq/apps/userreports/tests/../reports/builder/v1/forms.py
/mnt/commcare-hq-ro/corehq/apps/userreports/tests/../reports/builder/forms.py
**************************************
```

**Pull request merged**
